recipes/nginx.rb

```diff
@@ -16,17 +16,32 @@
 # limitations under the License.
 #
 
-if node['nagios']['server']['stop_apache']
-  service 'apache2' do
-    action :stop
+node.default['nagios']['server']['web_server'] = 'nginx'
+
+include_recipe 'chef_nginx'
+
+node.default['php-fpm']['pools']['www']['user'] = node['nginx']['user']
+node.default['php-fpm']['pools']['www']['group'] = node['nginx']['group']
+node.default['php-fpm']['user'] = node['nginx']['user']
+node.default['php-fpm']['group'] = node['nginx']['group']
+include_recipe 'php-fpm'
+
+package node['nagios']['server']['nginx_dispatch']['packages']
+
+if %w(rhel).include?(node['platform_family'])
```

Comment:
I prefer the %w( ).include? syntax because it allows me to easily add other values, amazon for example. I haven't had time to really investigate amazon (or fedora or freebsd).